docs: Add spectrogram parameter to Audio tag page (#7922)
heartexlabs/label-studio: docs/source/includes/tags/audio.md

| [sync] | <code>string</code> |  | Object name to sync with. |
| [height] | <code>string</code> | <code>96</code> | Total height of the audio player. |
| [waveheight] | <code>string</code> | <code>32</code> | Minimum height of a waveform when in `splitchannels` mode with multiple channels to display. |

> | [spectrogram] | <code>boolean</code> | <code>false</code> | Determines whether an audio spectrogram is automatically displayed instead of the audio waveform. |

Reviewer: ```suggestion
| [spectrogram] | <code>boolean</code> | <code>false</code> | Determines whether an audio spectrogram is automatically displayed upon loading. |
```
